lessons/lookups/langs/en-us/pages/defining-rows.adoc

```diff
@@ -16,46 +16,54 @@ dog-row = row-n(animals-table, 10) # Toggle is a dog
 
 **Remember: rows start at index zero!** Use this to answer the questions below.
 
-@n The index of a row containing a lizard is 					@right{@fitb{5em}{}}
-@n The index of a row containing a rabbit is 					@right{@fitb{5em}{}}
-@n The index of a row containing a fixed animal is 				@right{@fitb{5em}{}}
-@n The index of a row containing a male animal 					@right{@fitb{5em}{}}
-@n The index of a row containing a female animal is 			@right{@fitb{5em}{}}
-@n The index of a row containing a hermaphroditic animal is 	@right{@fitb{5em}{}}
-@n The index of a row containing an unfixed animal is 			@right{@fitb{5em}{}}
-@n The index of a row containing a young animal (<2 years) is 	@right{@fitb{5em}{}}
-@n The index of a row containing an old animal (>10 years) is 	@right{@fitb{5em}{}}
+@n The index of a row containing a lizard is @fitb{}{@ifsoln{21 or 22}}
```

Comment:
just out of curiosity - why did you remove the extra spaces to make the `fitb`s line up in the source file?

Reply:
This looked VERY WEIRD and wrong in the workbook. Like, the numbers that are supposed to go at the beginning of the line were showing up in the middle of the line, at the end of the line.... I was just trying to figure out what was going on. Ended up changing the `fitb` to extend across the whole line rather than 5em. Still not sure what the issue was?